csharp/Platform.Collections/Segments/Segment.cs

```diff
@@ -9,35 +9,73 @@
 
 namespace Platform.Collections.Segments
 {
+    /// <summary>
+    /// <para>Represents the segment <see cref="IList"/>.</para>
+    /// <para>Представляет сегмент <see cref="IList"/>.</para>
+    /// </summary>
+    /// <typeparam name="T"><para>The segment elements type.</para><para>Тип элементов сегмента.</para></typeparam>
     public class Segment<T> : IEquatable<Segment<T>>, IList<T>
     {
+        /// <summary>
+        /// <para>Returns the original list (of which this segment is a part).</para>
+        /// <para>Возвращает исходный список (частью которого является этот сегмент).</para>
+        /// </summary>
         public IList<T> Base
         {
             [MethodImpl(MethodImplOptions.AggressiveInlining)]
             get;
         }
+        /// <summary>
+        /// <para>Returns the offset of the relative source list (the index at which this segment starts).</para>
```

Comment:
`Gets the offset relative to the source list (the index at which this segment starts).`